Install/Chroot

```diff
@@ -191,29 +191,40 @@ Edit {{{/etc/fstab}}} file according to your disk setup in step №4:
 # nano /etc/fstab
 }}}
 
-* Compile a kernel (optional)
+* Update Sorcery and Scribe
 
-If you prefer to keep your system minimal, you might want to compile your own kernel with only needed hardware support and/or to include some advanced settings.
-You can do that just inside the chroot.
+** run {{{sorcery update}}} to update the sorcery scripts
+** run {{{scribe update && cast smgl-archspecs}}} to update the spell-lists and tell [[Sorcery]] the latest list for supported CPU architectures
+** configure Sorcery (run {{{sorcery}}}, then go to **Options**)
 
-There are 2 ways:
-# includes getting any desired image from http://www.kernel.org, extracting it to {{{/usr/src}}} (e.g.) and doing that by hand (via {{{make menuconfig}}}, {{{make}}}, {{{make modules}}}, copying bzImage, etc.)
-# includes typing:
+* Update Essential [[Spell|spells]] For Compiling
 
 {{{
+# sorcery hold g++
+# cast -c mpfr gmp
+# cast -c binutils gcc
+# sorcery unhold g++

```

Comment:
is g++ hack still relevant?

Reply:
probably not

Reply:
With the other items addressed, this appears to be the last one now that need to be looked at. As mentioned through the email thread, this is going to need a little bit of testing on the chroot machine I was using to see if the force depends function can be utilized as expected.

If this turns out to be true and force depends can be used, then I suspect we can drop this old work around and push what needed changes are required through our normal spell/grimoire update process.

I will circle back here once I have done some more testing.

Reply:
This has been brought up already through IRC, but I thought that I would mention it here for posterity. I setup a new chroot VM and went through the installation process. When I reached that appropriate step, I did not utilize the above work-around which was part of the pull request. I used the step originally noted in the ISO installation guide, which I presumed was before the work-around:

`# cast -c binutils gcc g++ gettext glibc`

After re-running the process three times, I was not able to find any issue with using the original update process instead of the work-around. As such, I will be updating this part of the chroot guide to reflect the original, non-work-around method.